mkdocs.yml

```
@@ -101,6 +101,7 @@ nav:
       - App Overview: "user/app_overview.md"
       - Getting Started: "user/app_getting_started.md"
       - Using the App: "user/app_use_cases.md"
+      - Integrations: "user/integrations.md"
```

```suggestion
      - Integrations: 
           - "user/integrations/index.md"
           - Cisco ACI: "user/integrations/aci.md"
```

As per other comment to structure folders.